src/perf.rs

```diff
@@ -13,7 +13,8 @@ struct PerfCallback {
     raw_cb: Box<dyn FnMut(&[u8]) + Send>,
 }
 
-const BPF_PERF_READER_PAGE_CNT: i32 = 64;
+/// This is a default safe page count to use when setting up the perf buffer
+pub const BPF_PERF_READER_PAGE_CNT: i32 = 64;
```

Comment:
It looks like the default is 8 in https://github.com/iovisor/bcc/blob/149c1c8857652997622fc2a30747a60e0c9c17dc/src/python/bcc/table.py#L664

How was 64 chosen for the default here?